django_short_urls/views.py

```diff
@@ -162,6 +162,10 @@ def new(request):  # pylint: disable=too-many-branches
                 status=HTTP_BAD_REQUEST,
                 message="%s may not contain a '/' character." % key)
 
+    statsd.increment(
+        'workforus.new',
+        tags=['prefix:' + unicode(params['prefix']), 'is_secure:' + unicode(request.is_secure())])
```

Comment:
Well, we're expecting 100% of a single value for `is_secure` so I'm not worried about multiplying tag combinations in this case, and adding the prefix will allow us to monitor which pages create the most `new` short links so I think it's quite useful.